base_module_record/models/base_module_record.py

```diff
@@ -127,7 +127,11 @@ def _create_record(self, doc, model, data, record_id, noupdate=False):
                 record.appendChild(field)
             elif fields[key]['type'] in ('one2many',):
                 for valitem in (val or []):
-                    if valitem[0] in (0, 1):
+                    if valitem[0] in (0, 1) and valitem[2].get(
+                            'name') not in ['create_date', 'create_uid',
```

Comment:
SPECIAL_FIELDS = []
use karo... we should make it flexible so to be able to allow it to be inherited